src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/missingoverride/Example1.java

```diff
@@ -0,0 +1,42 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MissingOverride"/>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.annotation.missingoverride;
+
+// xdoc section -- start
+class Example1{
+  public void test1(){}
+
+  public void test2(){}
+
+}
+
+class A extends Example1 {
+
+  /** {@inheritDoc} */
+  @Override
+  public void test1() { // OK
+
+  }
+
+  /** {@inheritDoc} */
+  public void test2() { // violation, 'include @java.lang.Override'
+
+  }
+
+  /** {@inheritDoc} */
+  private void test3() { // violation, 'The Javadoc {@inheritDoc}.'
+
+  }
+
+  /** {@inheritDoc} */
+  public static void test4() { // violation, 'The Javadoc {@inheritDoc}.'
```

Comment:
I do not understand this message.
https://github.com/checkstyle/checkstyle/blob/7a78eb4e3305e07108c50c0ff092f6043278769a/src/main/resources/com/puppycrawl/tools/checkstyle/checks/annotation/messages.properties#L17
please try to use more words from original message `{@inheritDoc} tag is not valid at this location.`

user should read your examples and understand them instantly.